.github/workflows/ci.yml

```diff
@@ -23,6 +23,11 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: "us-east-1"
 
+      - run: ./scripts/bootstrap
+        if: github.ref != 'refs/heads/master'
```

Comment:
For non-production environment, we want to run this step in order to use the corresponding config bucket to pull down the dotenv file. This is to prepare for the `cibuild` step.

I was not so sure about the deployment story of production, so I skipped uploading a dotenv file to the production bucket, and I also did not add an action here for prod either.

Reply:
We don't have a Phase II deployment task yet -- could you create one and add this as one of the required tasks?

Reply:
Good point. Done! https://github.com/azavea/echo-locator/issues/502